lib/ProductOpener/Ingredients.pm

```diff
@@ -895,12 +908,13 @@ fr => [
 ],
 
 cs => [
-'SLOŽENÍ',
+'SLOŽENÍ(\s*)(-|:|\r|\n)+',
 ],
 
 de => [
 'ZUTATEN(\s*)(-|:|\r|\n)+',	# need a colon or a line feed
 #@hangy Does that regex handle ZUTAT: ?
```

Comment:
@stephanegigandet can you confirm or not for `Zutat`  ?

Reply:
just change ZUTATEN by ZUTAT(EN)? and it will match both ZUTAT and ZUTATEN